src/js/components/Activity/ActivitySpeakerCard.jsx

```diff
@@ -71,31 +73,46 @@ class ActivitySpeakerCard extends Component {
     renderLog('ActivitySpeakerCard');  // Set LOG_RENDER_EVENTS to log all renders
     const {
       actionDescription, activityTimeFromDate, speakerIsVoter,
-      speakerName, // speakerOrganizationWeVoteId,
+      speakerName, speakerOrganizationWeVoteId,
       speakerProfileImageUrlMedium, speakerTwitterFollowersCount, speakerTwitterHandle,
     } = this.state;
     if (!speakerName) {
       return <div>{LoadingWheel}</div>;
     }
+    const organizationPopoverCard = (<OrganizationPopoverCard organizationWeVoteId={speakerOrganizationWeVoteId} />);
 
     // const voterGuideLink = speakerTwitterHandle ? `/${speakerTwitterHandle}` : `/voterguide/${speakerOrganizationWeVoteId}`;
 
     return (
       <Wrapper>
-        {(speakerProfileImageUrlMedium) ? (
-          <ProfileAvatar>
-            <ActivityImage src={speakerProfileImageUrlMedium} alt={`${speakerName}`} />
-          </ProfileAvatar>
-        ) : (
-          <ProfileAvatar>
-            <ActivityImage src={cordovaDot(avatarGenericIcon)} alt={`${speakerName}`} />
-          </ProfileAvatar>
-        )}
+        <StickyPopover
+          delay={{ show: 700, hide: 100 }}
+          popoverComponent={organizationPopoverCard}
+          placement="auto"
+          id="positions-organization-popover-trigger-click-root-close"
```

Comment:
Hi @mlevoy I will fix this, but id's are supposed to be unique when on the same page. I have also been shifting towards camelCase ids.

Reply:
woops, I just copy and pasted from the other file, my bad!